scripts/get-latestgithubrelease.ps1

```diff
@@ -0,0 +1,68 @@
+param(
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('win19', 'win22', 'ubuntu22', 'ubuntu20')]
+  [string]$OSTagPrefix
+)
+
+
+function Get-GitHubReleases {
+  param(
+    [string]$GitHubOwner,
+    [string]$RepoName
+  )
+
+  $apiUrl = "https://api.github.com/repos/$GitHubOwner/$RepoName/releases"
+  $headers = @{
+    'User-Agent' = 'PowerShell'
+  }
+
+  try {
+    $releases = Invoke-RestMethod -Uri $apiUrl -Headers $headers -Method Get
+    return $releases
+  }
+  catch {
+    Write-Host 'Error: Unable to retrieve GitHub releases.'
+    return $null
+  }
+}
+
+function Get-LatestGithubRelease {
+  param(
+    [string]$GitHubOwner = 'actions',
+    [string]$RepoName = 'runner-images',
+    [Parameter(Mandatory = $true)]
+    [ValidateSet('win19', 'win22', 'ubuntu22', 'ubuntu20')]
+    [string]$OSTagPrefix
+  )
+
+  $releases = Get-GitHubReleases -GitHubOwner $githubOwner -RepoName $repoName
+
+  $filteredreleases = @()
+  if ($releases) {
+    $releases | ForEach-Object {
+      if ($OSTagPrefix -eq $null -or $_.tag_name -like "*$OSTagPrefix*") {
```

Comment:
I think you only want to have a release that isn't a Pre-release. So it would be nice to have an extra filter here where you only get releases where `$_.prerelease -eq true`.

Reply:
@cengelenVW I agree, but I assume it will be `$_.prerelease -eq false`
I will have to try when reviewing what would be the best approach here, or potentially configurable between bleeding edge (like it is now), prerelease or released.

I love the effort indeed, because this brings it 95% there and saves me a ton of work (for which I sadly seem to be lacking time).

Reply:
Solved in last commit